### PR TITLE
[docs] updated README.md

### PR DESCRIPTION
**Pull request by windsonsea**
Fix some grammar and markdownlint issues.

Preview: https://github.com/cloudtty/cloudtty/blob/c0df98a54a10dd19489e8cbbccfe71843589f02a/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,148 +1,153 @@
-# A Kubernates Cloud-Shell (Web Terminal) Operator.
+# A Kubernates Cloud-Shell (Web Terminal) Operator
+
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/6301/badge)](https://bestpractices.coreinfrastructure.org/projects/6301)
 
 English | [简体中文](https://github.com/cloudtty/cloudtty/blob/main/README_zh.md)
 
-# Why needs cloudtty ?
+cloudtty is a kind of easy-to-use web terminal or cloud shell intended for a kubernetes-native environment. You can easily run the web tty as a pod on your own browser. The community is always open for any contributors and those who want to have a try.
```
Comment (panpan0000) [inline README.md:7]: `run the web tty as a pod on your own browser` is not accurate ...

Comment (windsonsea) [reply README.md:7]: How about this:
```
use the webtty as a pod on your own browser.
```
Not sure any more better wordings when I'm still in a very beginning understanding -_!

Comment (calvin0327) [reply README.md]: you can easily open a terminal on your own browser via cloudtty?

**Comment by calvin0327** [comment]
@windsonsea  Thank you for your contribution,  can you help to update the chinese documents synchronously?

**Comment by calvin0327** [comment]
@windsonsea  Thank you for your contribution,  can you help to update the chinese documents synchronously?

**Comment by windsonsea** [comment]
boys, men, guys, and masters, could you give the suggestion, not always ask questions when you review a doc?
1. Click + before a line 
2. Click the icon below
image
3. Directly modify the text, don't ask too many questions please.
4. Click `Start a review` if you have many comments. Click `Add single comment` if only one comment.

Thanks for efficiency.

**Comment by windsonsea** [comment]
> @windsonsea Thank you for your contribution, can you help to update the chinese documents synchronously?

Sure, that's my pleasure. Will handle if available.

File: README.md [hunk 2]
```diff
@@ -1,148 +1,168 @@
...
+# A Kubernates Cloud Shell (Web Terminal) Operator
+
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/6301/badge)](https://bestpractices.coreinfrastructure.org/projects/6301)
 
 English | [简体中文](https://github.com/cloudtty/cloudtty/blob/main/README_zh.md)
 
-# Why needs cloudtty ?
+cloudtty is an easy-to-use operator to run web terminal and cloud shell intended for a kubernetes-native environment.
+You can easily open a terminal on your own browser via cloudtty.
+The community is always open for any contributors and those who want to have a try.
```
Comment (panpan0000) [inline README.md:9]: `any contributor`

File: README.md [hunk 3]
```diff
@@ -1,148 +1,168 @@
...
+
+Early user terminals connected to computers were electromechanical teleprinters or teletypewriters (TeleTYpewriter, TTY), which might be the origin of TTY.
```
Comment (panpan0000) [inline README.md:11]: Is it too many words to explain `TTY` ?
or you can put a separated chapter with title `Why names it as TTY? `

Comment (windsonsea) [reply README.md]: Give a note to `what is tty`, which is searched out by google when I don't know. I'm a user of your cloudtty.

**Comment by calvin0327** [comment]
LGTM

@windsonsea thanks for your contribution

**Pull request merged**